runtime/polkadot/constants/src/lib.rs

```diff
@@ -111,6 +111,14 @@ pub mod xcm {
 	}
 }
 
+/// Parachains.
+pub mod parachains {
```

Comment:
```suggestion
/// System Parachains.
pub mod system_parachain_ids {
```

Reply:
I would have it as `system_parachain`, the constants has ID suffix

Reply:
Yeah that works too